test/dlc_tests/benchmark/ec2/pytorch/inference/test_performance_torch_inductor_inference.py

```diff
@@ -0,0 +1,177 @@
+# Copyright 2019-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+import os
+import time
+import pytest
+import re
+import pandas as pd
+import boto3
+import datetime
+from botocore.exceptions import ClientError
+from test.test_utils import (
+    CONTAINER_TESTS_PREFIX,
+    get_framework_and_version_from_tag,
+    LOGGER,
+    BENCHMARK_RESULTS_S3_BUCKET
+)
+
+SETUP_CMD = "cd $HOME && \
+             git clone --branch v2.0.0-rc3 --recursive --single-branch -depth 1 https://github.com/pytorch/pytorch.git && \
+             git clone --recursive https://github.com/pytorch/benchmark.git && \
+             git checkout $(cat pytorch/.github/ci_commit_pins/benchmark.txt && \
+             cd $HOME/benchmark && \
+             python install.py;"
+
+def unique_metric_dims(instance_type, precision, model_suite):
+    dimensions = [
+        {"Name": "InstanceType", "Value": instance_type},
+        {"Name": "ModelSuite", "Value": model_suite},
+        {"Name": "Precision", "Value": precision},
+        {"Name": "WorkLoad", "Value": "Inference"},
+    ]
+    return dimensions
+
+def get_boto3_session(region="us-east-1"):
+    """Get boto3 session with us-east-1 as default region used to connect to AWS services."""
+    return boto3.session.Session(region_name=region)
+
+def get_cloudwatch_client(region="us-east-1"):
+    """Get AWS CloudWatch client object. Currently assume region is IAD (us-east-1)"""
```

Comment:
Check env variable ``AWS_DEFAULT_REGION`` to get the current region

Reply:
this is default value, region is already passed from calling function